db-migration/migrate-measurement-collection-to-ttl.js

```diff
@@ -0,0 +1,16 @@
+db = connect(process.env.MONGODB_URL);
+
+const cursor = db.stations.find();
+while (cursor.hasNext()) {
+    const station = cursor.next();
+    const oldCollection = `${station._id}`;
+    const newCollection = `${station._id}-ttl`;
+
+    db[newCollection].createIndex({time: 1}, {expireAfterSeconds: 60 * 60 * 24 * 10});
+    db[oldCollection].aggregate([
+        {$set: {receivedAt: {$toDate: {$multiply: ["$time", 1000]}}}},
```

Comment:
`time` was already set as server received at time. Fortunately, the api is not using this field. Renaming to `receivedAt` and converting to a `datetime`.